specification/automation/resource-manager/Microsoft.Automation/stable/2015-10-31/definitions.json

```diff
@@ -1170,148 +797,39 @@
-            "name": "JobStatus",
-            "modelAsString": true
-          }
-        },
-        "statusDetails": {
-          "type": "string",
-          "description": "Gets or sets the status details of the job."
-        },
-        "startTime": {
-          "type": "string",
-          "format": "date-time",
-          "readOnly": true,
-          "description": "Gets the start time of the job.",
-          "x-nullable": true
-        },
-        "endTime": {
-          "type": "string",
-          "format": "date-time",
-          "readOnly": true,
-          "description": "Gets the end time of the job.",
-          "x-nullable": true
-        },
-        "exception": {
-          "type": "string",
-          "readOnly": true,
-          "description": "Gets the exception of the job."
-        },
-        "lastModifiedTime": {
+        "algorithm": {
           "type": "string",
-          "format": "date-time",
-          "readOnly": true,
-          "description": "Gets the last modified time of the job.",
-          "x-nullable": false
+          "description": "Gets or sets the content hash algorithm used to hash the content."
         },
-        "lastStatusModifiedTime": {
+        "value": {
           "type": "string",
-          "format": "date-time",
-          "readOnly": true,
-          "description": "Gets the last status modified time of the job.",
-          "x-nullable": true
-        },
-        "parameters": {
-          "type": "object",
-          "additionalProperties": {
-            "type": "string"
-          },
-          "description": "Gets or sets the parameters of the job."
-        }
-      },
-      "description": "Definition of Dsc Compilation job properties."
-    },
-    "DscCompilationJob": {
-      "properties": {
-        "properties": {
-          "$ref": "#/definitions/DscCompilationJobProperties",
-          "x-ms-client-flatten": true,
-          "description": "Gets or sets the properties of the Dsc Compilation job."
+          "description": "Gets or sets expected hash value of the content."
         }
       },
-      "allOf": [
-        {
-          "$ref": "#/definitions/ProxyResource"
-        }
+      "required": [
+        "algorithm",
+        "value"
       ],
-      "description": "Definition of the Dsc Compilation job."
+      "description": "Definition of the runbook property type."
     },
-    "DscCompilationJobListResult": {
+    "ContentLink": {
       "properties": {
-        "value": {
-          "type": "array",
-          "items": {
-            "$ref": "#/definitions/DscCompilationJob"
-          },
-          "description": "Gets or sets a list of Dsc Compilation jobs."
-        },
-        "nextLink": {
+        "uri": {
           "type": "string",
-          "description": "Gets or sets the next link."
-        }
-      },
-      "description": "The response model for the list job operation."
-    },
-    "DscConfigurationAssociationProperty": {
```

Comment:
Remove this property invalidates https://github.com/Azure/azure-rest-api-specs/pull/3190/files#diff-09777627978972db514896bb1119bc44R269

Reply:
Fixed.